modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/configuring-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-configuring-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure the {PM-kepler} to use the Redfish as the source for platform power.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*. If you already have a {PM-kepler} instance created, click *Edit Kepler*.
+
+. Configure {PM-kepler} instance’s `.spec.exporter.redfish` specifying the mandatory `secretRef` and optional `probeInterval` and `skipSSLVerify`.
```

Comment:
```suggestion
. Configure `.spec.exporter.redfish` of the {PM-kepler} instance by specifying the mandatory `secretRef` parameter. You can also configure the optional `probeInterval` and `skipSSLVerify` parameters to meet your needs.
```

The customer should decide if they want to configure anything optional. The step seems to indicate that the parameters/fields are needed.

Reply:
I really like your suggestion in here. Is it okay if I use "field" instead of parameters?

Reply:
For sure. Thanks!